apps/api/src/server/utils/application-factory-for-tests.js

```diff
@@ -96,6 +100,7 @@ const getApplicationDetails = ({ regions = true, mapZoomLevel = true, subSector
  *  description: string | null,
  *  caseStatus: string,
  *  dates?: {createdAt?: Date, modifiedAt?: Date, publishedAt?: Date},
+ *  gridReference?: {easting?: Number, northing?: Number}
```

Comment:
It doesn't seem like we are actually accepting easting and northing information in this function's arguments? We can probably remove this?